src/service/xhr-impl.js

```diff
@@ -189,8 +203,39 @@ export class Xhr {
       init.headers['Content-Type'] = 'application/json;charset=utf-8';
       init.body = JSON.stringify(init.body);
     }
-    return this.fetch(input, init)
-        .then(response => response.json());
+
+    const isCacheable = (init.method === 'GET' && propertyPath);
```

Comment:
I'm wondering: Is it a valid use case that you sometimes want the a sub path and sometimes the whole thing?